tools/chpl-language-server/test/test_server.py

```diff
@@ -0,0 +1,611 @@
+import sys
+import os
+import tempfile
+import typing
+import json
+
+from lsprotocol.types import ClientCapabilities
+from lsprotocol.types import CompletionList
+from lsprotocol.types import CompletionParams
+from lsprotocol.types import DefinitionParams
+from lsprotocol.types import DeclarationParams
+from lsprotocol.types import DidChangeWorkspaceFoldersParams
+from lsprotocol.types import ReferenceParams
+from lsprotocol.types import ReferenceContext
+from lsprotocol.types import InitializeParams
+from lsprotocol.types import LocationLink
+from lsprotocol.types import Location
+from lsprotocol.types import Position
+from lsprotocol.types import TextDocumentIdentifier
+from lsprotocol.types import WorkspaceFoldersChangeEvent
+from lsprotocol.types import WorkspaceFolder
```

Comment:
Can these imports be combined?

Reply:
We don't put all of them into one import statement in `chpl-language-server`, so I elected to match it here. I've grouped related imports, but still have several statements now.